packages/core/src/helpers/tests/empty.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai'
+
+import { feature, given } from '../../testing/ghrkn'
+import { isEmpty } from '../empty'
+
+feature`Empty helpers`(() => {
+  given`Non empty values expect value to be false`(() => {
+    [
+      true,
+      false,
+      ['I\'m not empty sir!'],
+      { hello: 'there' },
+      3,
+      'Im just empty string... UPS',
+      NaN,
+    ].forEach(empty => expect(isEmpty(empty)).to.be.false)
+  })
+  given`Non empty expect value to be true`(() => {
```

Comment:
given -> when -> then
given -> expect

I'd suggest adding `expect` helper to `gherkin` helpers

so it looks like:

```typescript
let emptyValues: any[];

given `some empty values` (() => {
  emptyValues = […]
})

expect `them to pass "isEmpty" test` (() => {
  emptyValues.forEach(v => {
     xpct(isEmpty(v)).to.be.true
  })
})
```

or

```typescript
let emptyValues: any[];
let testResults: boolean[];

given `some empty values` (() => {
  emptyValues = […]
})

when `they are tested with "isEmpty"` (() => {
  testResults = emptyValues.map(isSome) // or use filter and assert count
})

then `they all pass the test` (() => {
  expect(testResults).to.deep.equal([true, true, …])
})
```